crd/apis/network/v1/network_types.go

```diff
@@ -159,10 +159,36 @@ type Route struct {
 	To string `json:"to"`
 }
 
+// NetworkConditionType is the type for status conditions on
+// a Network. This type should be used with the
+// NetworkStatus.Conditions field.
+type NetworkConditionType string
+
+const (
+	// NetworkStatusReady is the condition type that holds
+	// if the Network object is validated
+	NetworkConditionStatusReady NetworkConditionType = "Ready"
```

Comment:
are we not having any reasons for Ready?

Reply:
I'll add them in this PR. 

Originally I was trying to break out my master changes for GNP controller which doesnt include the reasons for Network Ready.

Reply:
done